src/styles/utils/_variables

```diff
@@ -0,0 +1,12 @@
+$day-size: 100px;
```

Comment:
Don't forget to add extension of the file .scss. I see this file has no this extension.